modules/administration-guide/pages/editor-configurations-for-microsoft-visual-studio-code.adoc

```diff
@@ -73,6 +79,8 @@ immutable: false
 Make sure that the Configmap contains data in a valid JSON format.
 ====
 
+TIP: Consider adding the ConfigMap to the `eclipse-che` namespace. It allows to replicate the ConfigMap across all user namespaces while preventing modifications within user's namespaces. See xref:configuring-a-user-namespace.adoc[].
```

Comment:
`eclipse-che` is the upstream specific, we need to make sure that this part of the docs is properly downstreamed

Reply:
good catch!
thank you!

I replaced it by the `{prod-namespace}`

Reply:
if it looks good - will prepare a PR fix this place in the main branch